src/state/_store.ts

```diff
@@ -49,7 +49,7 @@ const persistConfig = {
   key: 'root',
   version: 2,
   storage,
-  whitelist: ['affiliates', 'dismissable', 'tradingView', 'wallet'],
+  whitelist: ['affiliates', 'dismissable', 'tradingView', 'wallet', 'configs'],
```

Comment:
cc @tinaszheng

Reply:
if we add `configs` I think the Abacus configs will also be stored. Maybe worth pulling `favorites` out of configs